tesseract/tesseract_collision/include/tesseract_collision/bullet/bullet_utils.h

```diff
@@ -476,13 +483,17 @@ inline void calculateContinuousData(ContactResult* col,
   // TODO: this section is potentially problematic. think hard about the math
   if (shape_sup0 - shape_sup1 > BULLET_SUPPORT_FUNC_TOLERANCE)
   {
+    // LCOV_EXCL_START
     col->cc_time[link_index] = 0;
     col->cc_type[link_index] = ContinuousCollisionType::CCType_Time0;
+    // LCOV_EXCL_STOP
```

Comment:
Why are these excluded?

Reply:
This is controlled by Bullet and I do not know of a good way create a condition where this gets triggered. I believe these are here to catch degenerate case of collision checking which are hard to create and in theory should not occur.